cmd/definitions/tmpl/service.tmpl

```diff
@@ -245,6 +248,33 @@ func parsePair{{ $pn }}{{ $fnk }}(opts []Pair) (pair{{ $pn }}{{ $fnk }}, error)
     }
     {{- end }}
 
+    // Default pairs
+    {{- range $_, $pair := $dp }}
+    if result.has{{ $pair.Name | toPascal }} {
+        result.HasDefault{{ $pn }}Pairs = true
+        {{- range $_, $op := $fu }}
+            {{- range $_, $opPair := $op.Required }}
+                {{- if eq $pair.Name (print "default_" $opPair.Name) }}
```

Comment:
Why? I thought we have `default_xxx` pairs already.

Reply:
I'm not so sure about your query.
The intention here is to find operations that contains the default pair, and the pairs for operations don't contain `default_` prefix.

Reply:
I'm a bit confused, let's discuss in https://matrix.to/#/#beyondstorage@gsp-700:matrix.org